Blockzilla/SettingsViewController.swift

```diff
@@ -357,23 +365,62 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
         let contentViewController = SettingsContentViewController(url: url)
         navigationController?.pushViewController(contentViewController, animated: true)
     }
-
+    
+    @objc func tappedLearnMoreSearchSuggestionsFooter(gestureRecognizer: UIGestureRecognizer) {
+        guard let url = SupportUtils.URLForTopic(topic: "search-suggestions") else { return }
+        let contentViewController = SettingsContentViewController(url: url)
+        navigationController?.pushViewController(contentViewController, animated: true)
+    }
+    
+    fileprivate func getToggleCell(indexPath: IndexPath) -> UITableViewCell {
+        let cell: UITableViewCell = UITableViewCell(style: .subtitle, reuseIdentifier: "toggleCell")
+        let toggle = toggleForIndexPath(indexPath)!
+        cell.textLabel?.text = toggle.label
+        cell.textLabel?.numberOfLines = 0
+        cell.accessoryView = PaddedSwitch(switchView: toggle.toggle)
+        cell.detailTextLabel?.text = toggle.subtitle
+        cell.detailTextLabel?.numberOfLines = 0
+        cell.selectionStyle = .none
+        if toggle.label == UIConstants.strings.labelSendAnonymousUsageData ||
+            toggle.label == UIConstants.strings.settingsSearchSuggestions{
+            let selector = toggle.label == UIConstants.strings.labelSendAnonymousUsageData ? #selector(tappedLearnMoreFooter) : #selector(tappedLearnMoreSearchSuggestionsFooter)
+            let learnMore = NSAttributedString(string: UIConstants.strings.learnMore, attributes: [.foregroundColor : UIConstants.colors.settingsLink])
+            let space = NSAttributedString(string: " ", attributes: [:])
+            guard let subtitle = toggle.subtitle else { return cell }
```

Comment:
Do this check at the beginning since a lot of the previous lines become unnecessary if it's just going to return.